### PR TITLE
Bump crate versions for publication

### PR DESCRIPTION
**Pull request by jethrogb**
Planning to release:

* rs-libc 0.2.1
* sgx-isa 0.3.2
* sgxs-loaders 0.2.2
* enclave-runner 0.4.0
* report-test 0.3.1
* dcap-ql 0.3.0
* aesm-client 0.5.0
* dcap-provider 0.3.1
* sgxs-tools 0.8.2
* fortanix-sgx-tools 0.4.0
* dcap-retrieve-pckid 0.1.0

(in that order)

### CODE REVIEW

**Comment by jethrogb** [comment]
bors try

File: enclave-runner/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "enclave-runner"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (mzohreva) [inline enclave-runner/Cargo.toml:3]: This should be a minor version bump due to tokio 0.2 stuff that landed recently.

Comment (jethrogb) [reply enclave-runner/Cargo.toml:3]: That change doesn't change the public API though, right?

Comment (mzohreva) [reply enclave-runner/Cargo.toml:3]: They change the return type of things in `UsercallExtension`

**Pull request merged**